modules/nf-core/happy/happy/main.nf

```diff
@@ -62,6 +62,7 @@ process HAPPY_HAPPY {
 
     stub:
     def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
     def VERSION = '0.3.14' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     """
     touch ${prefix}.summary.csv
```

Comment:
You'll need to use `echo "" | gzip > FILE` for gzipped files. Otherwise nf-test will fail